tools/release/publish_release.sh

```diff
@@ -404,6 +412,17 @@ tag() {
 
 publish() {
     if [ "$dry_run" = "false" ]; then
+        if [[ "$main_release" == "true" ]]; then
+            article_url="https://fleetdm.com/releases/fleet-$target_milestone"
+            article_published=`curl -is "$article_url" | head -n 1 | awk '{print $2}'`
+            if [[ "$article_published" != "200" ]]; then
+                echo "Coulndn't find article at '$article_url'"
+                exit 1
+            fi
```

Comment:
Do we need to manually merge JD's PR and wait for it to appear live on the website before running the script? That works for me, I just want to be clear.